packages/integration-tests/src/tests/api/custom-phrase.test.ts

```diff
@@ -0,0 +1,84 @@
+import { HTTPError } from 'got';
+
+import {
+  listCustomPhrases,
+  getCustomPhrase,
+  createOrUpdateCustomPhrase,
+  deleteCustomPhrase,
+} from '#src/api/custom-phrase.js';
+
+const mockZhTranslation = {
+  input: {
+    email: '邮箱',
+    password: '密码',
+    username: '用户名',
+    phone_number: '手机号码',
+    confirm_password: '确认密码',
+    search_region_code: '搜索区号',
+  },
+};
+
+const mockZhTranslationUpdated = {
+  ...mockZhTranslation,
+  description: {
+    email: '邮箱',
+    phone_number: '手机号码',
+  },
+};
+
+const mockEnUsTranslation = {
+  input: {
+    email: 'email',
+    password: 'password',
+    username: 'username',
+    phone_number: 'phone_number',
+    confirm_password: 'confirm password',
+    search_region_code: 'search region code',
+  },
+};
+
+describe('custom-phrase flow', () => {
```

Comment:
nit: test case for updating with an invalid prase key?

Reply:
Added, also a string typed `languageTag` make more sense. Also updated that.